Monika After Story/game/zz_selector.rpy

```diff
@@ -2917,9 +3062,43 @@ screen mas_selector_sidebar(items, mailbox, confirm, cancel, restore, remover=No
 #    modal True
 
     $ sel_frame_vsize = mailbox.read_frame_vsize()
+    default flt_items = items
+
+    # Search bar
+    frame:
+        xpos 1075
+        ypos 5
+        xsize 200
+        ysize 40
+        background Solid("#ffaa99aa")
+
+        viewport:
+            draggable False
+            arrowkeys False
+            mousewheel "horizontal"
+            # I have no idea why, but this must be 5 px shorter
+            xsize 195
+            ysize 38
+            xadjustment ui.adjustment(ranged=store.mas_selspr.selector_adj_ranged_callback)
+
+            input:
+                id "search_input"
+                style_prefix "input"
+                length 50
+                xalign 0.0
+                layout "nobreak"
+                # allow "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789 _"
+                changed store.mas_selspr.selector_search_callback
+
+        # NOTE: we do need an instance check here
+        if flt_items is items:
+            text "Search for...":
+                text_align 0.0
+                layout "nobreak"
+                color "#FFFFFF"
```

Comment:
Tried some style changes and was satisfied with this. This makes the ghost text look more like real ghost text.
```suggestion
                color "#EEEEEEB2"
                first_indent 10
                line_leading 1
                outlines []
```